src/models/AppType.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * @flow
+ */
+
+import 'immutable';
+import Logger from '../utils/Logger';
+
+import {
+  isDefined,
+  isNonEmptyString
+} from '../utils/LangUtils';
+
+import { isValidUuid } from '../utils/ValidationUtils';
+import FullyQualifiedName from './FullyQualifiedName';
+
+const LOG = new Logger('AppType');
+
+/**
+ * @class AppType
+ * @memberof lattice
+ */
+export default class AppType {
```

Comment:
You're missing the "id" field.

https://github.com/openlattice/api/blob/develop/src/main/java/com/openlattice/apps/AppType.java#L19